zproject/local_settings.py

```diff
@@ -59,6 +59,9 @@
     EXTERNAL_API_PATH = 'api.zulip.com'
     STATSD_PREFIX = 'app'
 
+# Terms of Service
+CUSTOM_TOS = '/etc/zulip/terms.md'
```

Comment:
could this be instead just `/home/zulip/deployments/current/templates/corporate/terms.md` ?  That would save a migration to copy this file into place for the zulip.com folks deploying this...